TUTORIAL.md

```diff
@@ -61,25 +61,25 @@ This information is used by the main server thread to maintain a list of active
 
 #### Connecting to Ably
 
-In order to use this kit, you will need an Ably API key. If you are not already signed up, you can [sign up now for a free Ably account](https://www.ably.io/signup). Once you have an Ably account:
+In order to use this kit, you will need an Ably API key. If you are not already signed up, you can [sign up now for a free Ably account](https://www.ably.com/signup). Once you have an Ably account:
 
 - Log into your app dashboard
 - Under **Your apps**, click on **Manage app** for any app you wish to use for this tutorial, or create a new one with the **Create New App** button
 - Click on the **API Keys** tab
 - Copy the secret **API Key** value from your Root key.
 
-The server-side scripts connect to Ably using [Basic Authentication](https://www.ably.io/documentation/core-features/authentication#basic-authentication), i.e. by using the API Key directly as shown below:
+The server-side scripts connect to Ably using [Basic Authentication](https://www.ably.com/documentation/core-features/authentication#basic-authentication), i.e. by using the API Key directly as shown below:
 
 ```js
-const realtime = Ably.Realtime({
+const realtime = new Ably.Realtime({
   key: ABLY_API_KEY,
```

Comment:
I don't see any use of the `env` file that contains the Ably key in this tutorial. Is this on purpose?

Reply:
very good catch. thanks.